test/helpers/lsp.cc

```diff
@@ -10,23 +10,21 @@
 namespace sorbet::test {
 using namespace std;
 
-string filePathToUri(string_view prefixUrl, string_view filePath) {
-    return fmt::format("{}/{}", prefixUrl, filePath);
+string filePathToUri(const LSPConfiguration &config, string_view filePath) {
+    return fmt::format("{}/{}", config.getClientConfig().rootUri, filePath);
 }
 
-bool isUriATestFile(string_view prefixUrl, string_view uri) {
```

Comment:
This method is no longer needed -- `LSPConfiguration` defines a more robust version.